app/admin/2fa/index.php

```diff
@@ -161,16 +161,16 @@
 
 
 <article style='margin-top:50px;' class="text-muted">
-phpIPAM supports two-factor-authentication to add additional security layer for user authentication.
+<?php print _("phpIPAM supports two-factor-authentication to add additional security layer for user authentication.".); ?>
 <br>
-After user successfully logs in it will be presented with additional screen to enter code from Google authenticator application.
+<?php print _("After user successfully logs in it will be presented with additional screen to enter code from Google authenticator application.".); ?>
 <br><br>
-Google Authenticator is available on following links based on your OS:
+<?php print _("Google Authenticator is available on following links based on your OS:".); ?>
 <ul>
 	<li> <a href='https://itunes.apple.com/us/app/google-authenticator/id388497605?mt=8' target="_self">Apple iOS</a></li>
 	<li> <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2" target="_self">Android</a></li>
 </ul>
-You can also use Microsoft or any other otp provider.
+<?php print _("You can also use Microsoft or any other otp provider.".); ?>
```

Comment:
There are a few syntax errors here, extra full stop before the closing bracket.  `".); `

``` php
[root@centos7 ~]# php -a
Interactive shel
l
php > print _("You can also use Microsoft or any other otp provider.".);
PHP Parse error:  syntax error, unexpected ')' in php shell code on line 1
php >
```

Reply:
meeh, what a stupid oversight! Should be corrected now.